Add Karma package (#143)
sighupio/fury-kubernetes-monitoring: katalog/karma/README.md

# Karma

<!-- <KFD-DOCS> -->

Karma is a UI for Alertmanager, useful for browsing alerts based on labels and managing silences.
It can also aggregate alerts from multiple Alertmanager instances.

*Source:* [prymitive/karma][k-gh]

## Requirements

- Kubernetes >= `1.21.0`
- Kustomize `= v3.5.3`
- [prometheus-operator](../prometheus-operator)
- [prometheus-operated](../prometheus-operated)
- [alertmanager-operated](../alertmanager-operated)

## Image repository and tag

- Karma image: `registry.sighup.io/fury/karma/karma:v0.113`
- Karma repository: [Karma on GitHub][k-gh]

## Configuration

Fury distribution Karma is deployed with the following

> configuration:
> - Alertmanager URI: `http://alertmanager-main.monitoring.svc.cluster.local:9093`
> - Polling interval: `1m`

Reviewer: linter will complain if you don't leave an empty line before a list
```suggestion
configuration:

- Alertmanager URI: `http://alertmanager-main.monitoring.svc.cluster.local:9093`
- Polling interval: `1m`
```

> All configuration options can be found [here](https://github.com/prymitive/karma/blob/main/docs/CONFIGURATION.md)

Reviewer: I suggest linking to the docs for the version we are using:

```suggestion
All configuration options can be found [here](https://github.com/prymitive/karma/blob/v0.113/docs/CONFIGURATION.md)
```

pointing to `main` has the downside that the file contents could change or even disappear :)

> ## Deployment
> You can deploy karma by running the following command:

Reviewer: linting
```suggestion
## Deployment

You can deploy karma by running the following command:
```

maybe we can say something about how to create the configuration patch here?
